code/__DEFINES/misc.dm

```diff
@@ -253,7 +253,8 @@
 #define MATRIX_DEFAULT list(1, 0, 0, 0,\
                             0, 1, 0, 0,\
                             0, 0, 1, 0,\
-                            0, 0, 0, 1)
+                            0, 0, 0, 1,\
+                            0, 0, 0, 0)
```

Comment:
> the logic that determines whether a person's client.colour == or != whatever the colour matrix that's being applied

Yeah, but you don't need that logic, since you should always be clobbering the old color anyway.